sretoolbox/container/image.py

```diff
@@ -29,9 +29,23 @@
 
 _LOG = logging.getLogger(__name__)
 
-MANIFEST_MEDIA_TYPE = 'application/vnd.docker.distribution.manifest.v2+json'
-MANIFEST_LIST_MEDIA_TYPE = \
+SCHEMA1_MANIFEST_MEDIA_TYPE = \
+    'application/vnd.docker.distribution.manifest.v1+json'
+SCHEMA1_SIGNED_MANIFEST_MEDIA_TYPE = \
+    'application/vnd.docker.distribution.manifest.v1+prettyjws'
+SCHEMA2_MANIFEST_MEDIA_TYPE = \
+    'application/vnd.docker.distribution.manifest.v2+json'
+SCHEMA2_MANIFEST_LIST_MEDIA_TYPE = \
     'application/vnd.docker.distribution.manifest.list.v2+json'
+OCI_MANIFEST_MEDIA_TYPE = 'application/vnd.oci.image.manifest.v1+json'
+OCI_IMAGE_INDEX_MEDIA_TYPE = 'application/vnd.oci.image.index.v1+json'
+
+SINGLE_ARCH_MEDIA_TYPES = [SCHEMA2_MANIFEST_MEDIA_TYPE,
+                           OCI_MANIFEST_MEDIA_TYPE]
+MULTI_ARCH_MEDIA_TYPES = [SCHEMA2_MANIFEST_LIST_MEDIA_TYPE,
+                          OCI_IMAGE_INDEX_MEDIA_TYPE]
+
+NO_SUCH_HEADER = "No such header!?!?!"
```

Comment:
lol loved it, but I I don't think it is necessary

Reply:
fixed in https://github.com/app-sre/sretoolbox/commit/b58b09be4a6c06a486afa871c6da3e810f22df98